internal/datacoord/handler.go

```diff
@@ -206,14 +207,35 @@ func (h *ServerHandler) GetQueryVChanPositions(channel RWChannel, partitionIDs .
 	// unindexed is flushed segments as well
 	indexedIDs.Insert(unIndexedIDs.Collect()...)
 
+	// update partition stats versions
```

Comment:
when there are too many partitions and multiple versions of partition stats are avaiable to single partition, this part can consume loads of cpu cycles, no mentioning it includes string prefix compare